components/centos/centos-mesos-marathon-singlenode-setup/Vagrantfile

```diff
@@ -4,9 +4,14 @@
 
 IP_ADDRESS="10.2.2.2"
 
-#check if vagrant-service-manager is installed on system
-unless Vagrant.has_plugin?("vagrant-service-manager")
-  raise "vagrant-service-manager plugin is not installed, run `vagrant plugin install vagrant-service-manager` to install the plugin."
+REQUIRED_PLUGINS = %w(vagrant-service-manager vagrant-registration)
+message = ->(name) { "#{name} plugin is not installed, run `vagrant plugin install #{name}` to install it." }
```

Comment:
Can this get moved into the function to increase readability of this file?

One day this is a candidate for the "in the box" vagrantfile ...